### PR TITLE
Adds basic interactions [wip]

### PR DESCRIPTION
**Pull request by caseyyee**
Getting some basic interaction scripts in place to better show usage.

### CODE REVIEW

File: Assets/WebVR/Scripts/Interaction.cs [hunk 1]
```diff
@@ -0,0 +1,61 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Interaction : MonoBehaviour {
```
Comment (cvan) [inline Assets/WebVR/Scripts/Interaction.cs]: nice! did you write this from scratch? or adapted from somewhere?

Comment (caseyyee) [reply Assets/WebVR/Scripts/Interaction.cs]: parts of it were from VIVE tutorials I found around.

**Comment by caseyyee** [comment]
physics backwards.

Due to this bug:
https://bugzilla.mozilla.org/show_bug.cgi?id=1362545

**Comment by cvan** [comment]
I can't tell what's changed here. the scene looks the same to me.

what about the Robots example (that used to live at http://objective-tesla-811b7f.bitballoon.com/)?

**Comment by caseyyee** [comment]
@cvan you'll need to build before trying.   Still WIP, will be hacking on it more this week to get it into a more useable state.

**Comment by cvan** [comment]
> @cvan you'll need to build before trying. Still WIP, will be hacking on it more this week to get it into a more useable state.

I did build it. the scene didn't appear any differently. are some file changes not commited to this branch or something else missing?

**Comment by cvan** [comment]
@caseyyee can you rebase this to the latest changes? also, what's left to merge this? anything major in particular broken or needed testing?

**Comment by caseyyee** [comment]
@cvan working on this now.  will rebase.  Just want to fix a few hacks.

**Comment by caseyyee** [comment]
Going to split this into smaller changes and PR's.

**Pull request closed without merging**